### PR TITLE
feat(bootstrap) Enable bootstrap straight from help scout conversation

### PR DESCRIPTION
**Pull request by s-pace**

### CODE REVIEW

File: deployer/src/config_creator.py [hunk 1]
```diff
@@ -25,6 +72,17 @@ def create_config():
     u = helpers.get_user_value("start url: ")
     urls = [u]
 
+    if 'secure.helpscout.net/conversation' in u:
+        cuid=getConversationIDfromUrl(u)
```
Comment (maxiloc) [inline deployer/src/config_creator.py]: => `cuid = getConversationIDfromUrl(u)`

File: deployer/src/config_creator.py [hunk 2]
```diff
@@ -25,6 +72,17 @@ def create_config():
...
+
+        conversations = getConversations(cuid)
+        url_from_conversations = getStartUrlFromConversations(conversations)
+
+        config["conversations_id"]=[cuid]
```
Comment (maxiloc) [inline deployer/src/config_creator.py]: same here we should add space to the equal

File: deployer/src/config_creator.py [hunk 3]
```diff
@@ -2,9 +2,56 @@
...
+def getConversationIDfromUrl(hs_url):
+    capture_conversation_uid = re.compile('.+/conversation/(\d+)/.*')
+    cuid = capture_conversation_uid.match(hs_url).group(1)
+    if not len(cuid) > 0:
+        raise ValueError("Wrong help scout url " + hs_url + ", must have a conversation sub part with ID")
+    if not RepresentsInt(cuid):
+        raise ValueError("Conversation ID : " + cuid + " must be an integer")
+    return cuid
+
+def getConversations(cuid):
```
Comment (maxiloc) [inline deployer/src/config_creator.py]: I would make all conversation related name singular

**Comment by s-pace** [comment]
PTA @maxiloc

**Pull request merged**